test/trento_web/views/v1/activity_log_view_test.exs

```diff
@@ -60,7 +22,7 @@ defmodule TrentoWeb.V1.ActivityLogViewTest do
              metadata: ^metadata,
              occurred_on: ^inserted_at
            } =
-             render(ActivityLogView, "activity_log_entry.json", %{
+             ActivityLogView.render("activity_log_entry.json", %{
```

Comment:
I think it is more idiomatic the way it is. With what you are suggesting, it would require an additional Phoenix View import and using that as a dispatching mechanism adds more indirection than is strictly needed.

Reply:
The most important thing is we keep the codebase coherent and consistent. The same instructions must be written with the same shape so that we avoid having a scattered codebase in which each developer feels free to use his/her taste regardless of the others.

If it brings an advantage we can refactor all the occurrences. I advocate we open a PR on that.
‏

Reply:
Consistency is always nice. Not so sure if this introduces any incoherence. Regardless, if you see this as an opportunity for process improvement, please feel free to add a custom Credo consistency check to the CI for this/similar patterns. That would be the way to ensure that a shared state of consistency is evenly followed everywhere.